kolibri/core/auth/constants/collection_kinds.py

```diff
@@ -8,10 +8,12 @@
 FACILITY = "facility"
 CLASSROOM = "classroom"
 LEARNERGROUP = "learnergroup"
+INDIVIDUALLEARNERSGROUP = "individuallearnersgroup"
```

Comment:
Is the function of this new collection kind:

1. A special "collection" that really should always have exactly 1 user in it
2. A new "collection" that is not shown to coaches, but represents an ad-hoc group of individual learners within a class?

Reply:
This collection creates a new kind of group.

- It is a collection that should always be assigned to one exam.
- It can have any number of learners in it who are also associated with the parent collection (classroom).
- It can be shown at will on the front-end. Currently, it is only shown to Coaches while in the Quiz > Edit Details page but it could relatively easily be included into the Lessons if desired down the road.
- It is not shown as a "group" in the Groups page at all, though if that's what you mean by "not shown to coaches"